frappe/integrations/doctype/google_drive/google_drive.json

```diff
@@ -0,0 +1,119 @@
+{
+ "creation": "2019-08-13 17:24:05.470876",
+ "doctype": "DocType",
+ "engine": "InnoDB",
+ "field_order": [
```

Comment:
Add more options like we have in Dropbox Settings:

![image](https://user-images.githubusercontent.com/9355208/63161996-1b007c00-c03f-11e9-9a5f-15a03bf397ad.png)